components/expenses/ExpenseNeedsTaxFormBadge.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import Error from '../Error';
+
+const getIsTaxFormRequiredQuery = gql`
```

Comment:
On the host dashboard where we show 20 expenses this triggers 20 additional fetchs. As outlined in https://github.com/opencollective/opencollective-api/pull/2074#discussion_r295665752 the endpoint itself can trigger up to 3 SQL queries so we end up with 20-60 additional SQL queries just to load the host dashboard, that's a lot. Any chance we could improve that?

Reply:
> Any chance we could improve that?

First, I'd love to have numbers for how much longer this actually takes to render 20 expenses. I'm nervous of premature optimisation unless it's needed. Do you have an intuition about how long each query takes? 

If it is needed or it's really something that's a blocker for you, that's fine but I'd love some guidance on how to do this well. If it's using the loaders, could you give me some pointers?

Reply:
It's about having a clear pattern. Only a limited number of people will query this field so I'm not worried so much about the performance impact.

Having a list with each item fetching its own data is something that has been done in the past for the host dashboard but we should definitely avoid that, it opens the door for N+1 queries. I don't think that avoiding N+1 queries is premature optimization.

Without spending time on a loader, we could already avoid that for the frontend by fetching the info at the top level, for example in `ExpensesWithData`. That way we don't need additional queries to retrieve the info.

Reply:
Ok, I've moved the query into `ExpensesWithData` as you suggest.